book/src/external-snapshotter.md

```diff
@@ -10,13 +10,15 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](project-policies.md#minimum-version) | [Max K8s Version](project-policies.md#maximum-version) | [Recommended K8s Version](project-policies.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-snapshotter v6.3.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v6.3.0) | [release-6.2](https://github.com/kubernetes-csi/external-snapshotter/tree/release-6.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1 | v1.20 | - | v1.24
-[external-snapshotter v6.2.2](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v6.2.2) | [release-6.2](https://github.com/kubernetes-csi/external-snapshotter/tree/release-6.2) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1 | v1.20 | - | v1.24
+[external-snapshotter v8.0.1](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v8.0.1) | [release-8.0](https://api.github.com/repos/kubernetes-csi/external-snapshotter/tree/release-v8.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | [v1.9.0](https://github.com/container-storage-interface/spec/releases/tag/v1.9.0) | registry.k8s.io/sig-storage/csi-snapshotter:v8.0.1 | v1.25 | - | v1.25 |
 
 ### Unsupported Versions
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](project-policies.md#minimum-version) | [Max K8s Version](project-policies.md#maximum-version) | [Recommended K8s Version](project-policies.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[external-snapshotter v7.0.2](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v7.0.2) | [release-7.0](https://api.github.com/repos/kubernetes-csi/external-snapshotter/tree/release-v7.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | [v1.9.0](https://github.com/container-storage-interface/spec/releases/tag/v1.9.0) | registry.k8s.io/sig-storage/csi-snapshotter:v7.0.2 | v1.20 | - | v1.24 |
```

Comment:
v7.0.0 was released in January this year: https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v7.0.0. So we need to support 7.0.x until Jan next year.